docs/time-varying-infectiousness.md

```diff
@@ -0,0 +1,205 @@
+[order statistics](https://en.wikipedia.org/wiki/Order_statistic)", and it is readily solvable for the
+case of the uniform distribution.
+
+## Drawing ordered samples from a uniform distribution
+
+Let us begin by considering the minimum of $n$ draws from a uniform distribution. We denote the sorted
+first of these draws as $X_{(1)}$, the sorted second as $X_{(2)}$, etc. Note that this is different from $X_1$
+which is the unsorted first random draw from the distribution, unsorted. We are interested in the distribution of
+$X_{(1)}$. Let us consider the cumulative distribution function of $X_{(1)}$ since it is a continuous random variable.
+
+$$\mathbb{P} \\{X_{(1)} \leq x \\}$$
+
+We know that if the minimum is below some value, $x$, that could mean that just the minimum is below $x$ or that
+all $n$ of our random samples are below the minimum. There are too many values to enumerate, so let us consider
+the opposite instead -- that $X_{(1)}$ is greater than some value $x$.
+
+$$\mathbb{P} \\{X_{(1)} \leq x \\} = 1 - \mathbb{P} \\{X_{(1)} > x \\}$$
+
+In this case, if $X_{(1)} > x$, we know that all of sampled values are at least above $x$. Recall that
+each of the samples is independent and identically-distributed.
+
+$$= 1 - \mathbb{P} \\{X_i > x \\}^n$$
+
+$$= 1 - (1 - F_X(x))^n$$
+
+For a uniform distribution:
+
+$$= 1 - (1 - x)^n$$
+
+This is the CDF for a Beta distribution with $alpha = 1$ and $beta = n$. More generally, the distribution
+of the $k$th infection attempt from $n$ total infection attempts is $\beta(k, n - 1 + k)$.
+
+However, we cannot just independently sample from these Beta distributions. Instead, we must update the distributions
+to be conditioned on the previously drawn times to ensure that our infection attempt times are always increasing.
+In other words, if we draw the first infection attempt time from $\beta(1, 5)$ and happen to get a large value, we need
+to take that into account when drawing the second infection attempt time. It must be greater than the first
+value we drew, so we cannot just independently take a draw from $\beta(2, 6)$ but rather $\beta(2, 6) | x_{(1)}$.
+
+We can consider that the second infection time is not independent from the first, the third not independent from
+the first and the second, etc. by rephrasing the problem. Once we have taken our first infection attempt, $x_{(1)}$,
+we know that any subsequent infection attempt times must be greater. We can set $x_{(1)}$ as the minimum of a new uniform
+distribution, $\mathcal{U}(x_{(1)}, 1)$, from which we need to draw an infection attempt. Because this is a new distribution,
+we want the first of $n - 1$ infection attempt times on this distribution. We can do that by drawing the
+minimum of $n - 1$ infection attempts from $\mathcal{U}(0, 1)$, and scaling that value to be on $(x_{(1)}, 1)$.
+In other words, we are using a trick where we shrink the available uniform distribution with each infection
```

Comment:
I would delete this "we are using a trick where", and just say what we are doing.